R/metrics.r

```diff
@@ -200,15 +201,15 @@ gap_fraction_profile = function (z, dz = 1)
 #' @references Bouvier, M., Durrieu, S., Fournier, R. a, & Renaud, J. (2015).  Generalizing predictive models of forest inventory attributes using an area-based approach with airborne LiDAR data. Remote Sensing of Environment, 156, 322-334. http://doi.org/10.1016/j.rse.2014.10.004
 #' @seealso \link[lidR:gap_fraction_profile]{gap_fraction_profile}
 #' @export LAD
-LAD = function(z, dz = 1, k = 0.5) # (Bouvier et al. 2015)
+LAD = function(z, dz = 1, k = 0.5, z0 = 2) # (Bouvier et al. 2015)
 {
 	ld = gap_fraction_profile(z, dz)
 
 	if(anyNA(ld))
 	  return(NA_real_)
 
 	lad = ld$gf
-	lad = -log(lad)/k
+	lad = -log(lad)/(k*dz)
```

Comment:
Ok, devtools::document(), update NEWS.md, devtools::check() done
I think I made the new branch from master, that is why it had a conflict with devel in NEW.md, resolved it.